index.html

```diff
@@ -23,14 +23,149 @@
+        </article>
+
+          <img src="images/Rectangle 21v2.png" width="100%">
+          <article class="works">
+          <h3>Project 3</h3>
+        
+          <ul class="technology-container">
+            <li class="item1">Html/Css</li>
+            <li class="item2">Ruby on rails</li>
+            <li class="item3">JavaScript</li>
+          </ul>
+          <div class="item4">
+            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+              <i class="see-project"></i>See this project <img src="images/Union.png" alt="arrow-right"></a>
+          </div>
+        </article>
+
+        
+          <img src="images/Rectangle 21 (1).png" width="100%">
+          <article class="works">
+          <h3>Project 4</h3>
+        
+          <ul class="technology-container">
+            <li class="item1">Html/Css</li>
+            <li class="item2">Ruby on rails</li>
+            <li class="item3">JavaScript</li>
+          </ul>
+          <div class="item4">
+            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+              <i class="see-project"></i>See this project <img src="images/Union.png" alt="arrow-right"></a>
+          </div>
+        </article>
+
+          <img src="images/Rectangle 21 (2).png" width="100%">
+          <article class="works">
+          <h3>Project 5</h3>
+        
+          <ul class="technology-container">
+            <li class="item1">Html/Css</li>
+            <li class="item2">Ruby on rails</li>
+            <li class="item3">JavaScript</li>
+          </ul>
+          <div class="item4">
+            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+              <i class="see-project"></i>See this project <img src="images/Union.png" alt="arrow-right"></a>
+          </div>
+        </article>
+
+          <img src="images/Rectangle 21 (3).png" width="100%">
+          <article class="works">
+          <h3>Project 6</h3>
+        
+          <ul class="technology-container">
+            <li class="item1">Html/Css</li>
+            <li class="item2">Ruby on rails</li>
+            <li class="item3">JavaScript</li>
+          </ul>
+          <div class="item4">
+            <a href="https://github.com/Maedea9/Project-mobile-first" class="button">
+              <i class="see-project"></i>See this project <img src="images/Union.png" alt="arrow-right"></a>
+          </div>
+        </article>
+        </section>
+
+        <section class="aboutme">
+          <h2>About</h2>
+          <p class="second-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+            aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod.</p>
+        
+        <div class="box">
+          <a href="#" class="button1">
+            <i class="download"></i><img src="images/Group 86.png" alt="download-icon"></a>
```

Comment:
- [ ] [OPTIONAL] From the design, I think the only image here is the `download icon`, and not the entire button text. It will feel a lot nicer and more convenient to have your text handwritten in case you want to change its color later in the future :wink: